.readthedocs.yaml

```diff
@@ -33,4 +33,4 @@ build:
       - make setup-docs UV_PROJECT_ENVIRONMENT="${READTHEDOCS_VIRTUALENV_PATH}"
     build:
       html:
-        - make docs UV_PROJECT_ENVIRONMENT="${READTHEDOCS_VIRTUALENV_PATH}" DOCS_OUTPUT="$READTHEDOCS_OUTPUT/html/"
+        - make docs UV_PROJECT_ENVIRONMENT="${READTHEDOCS_VIRTUALENV_PATH}" DOCS_OUTPUT="$READTHEDOCS_OUTPUT/html/" GIT_DISCOVERY_ACROSS_FILESYSTEM=1
```

Comment:
@mr-cal This brings us to:

```
Error getting data from Git (no "last updated" dates will be shown for source files from /home/docs/checkouts/readthedocs.org/user_builds/canonical-snapcraft/envs/5579/lib/python3.12/site-packages/craft_parts_docs/craft-parts/explanation):
fatal: not a git repository (or any of the parent directories): .git
```

The last line is a departure, but I don't know why it's printing `.git`.

The build is acting like `git_exclude_patterns` has no effect. I could dig into this further, but I don't know whether it's worth it. The `lastmod`s add a modest amount of value to the sitemap.